apollo-portal/src/main/resources/static/scripts/directive/release-modal-directive.js

```diff
@@ -59,23 +59,23 @@ function releaseModalDirective($translate, toastr, AppUtil, EventManager, Releas
                     scope.toReleaseNamespace.releaseTitle,
                     scope.releaseComment,
                     scope.isEmergencyPublish).then(
-                        function (result) {
```

Comment:
please keep the origin format to avoid unnecessary changes

Reply:
ok,I will do it.Thank you